src/xcode/ENA/ENASecurity/Sources/ENASecurity/Model/JSONWebKey.swift

```diff
@@ -6,7 +6,7 @@ import Foundation
 import ASN1Decoder
 
 public struct JSONWebKey: Codable {
-    let x5c: [String]
+    public let x5c: [String]
```

Comment:
does this has to be `public` explicit or is `internal` not "enough"?

Reply:
It needs to be checked if it is empty or not as part of the validation inside the ticketValidation class, that is why it needs to be public as the package is not accessible from the app itself